dependencies/bom/pom.xml

```diff
@@ -278,6 +278,11 @@
         <artifactId>registry-zookeeper</artifactId>
         <version>${project.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.servicecomb</groupId>
+        <artifactId>registry-etcd</artifactId>
+        <version>${project.version}</version>
+      </dependency>
```

Comment:
sort by name

Reply:
I don't understant what your mean?  Is it sorted by the name of artifactId?